bundles/core/src/main/java/com/adobe/cq/commerce/core/components/models/searchresults/SearchResults.java

```diff
@@ -11,22 +11,62 @@
  *    governing permissions and limitations under the License.
  *
  ******************************************************************************/
-
 package com.adobe.cq.commerce.core.components.models.searchresults;
 
 import java.util.Collection;
+import java.util.List;
 
 import javax.annotation.Nonnull;
 
+import org.osgi.annotation.versioning.ProviderType;
+
 import com.adobe.cq.commerce.core.components.models.common.ProductListItem;
+import com.adobe.cq.commerce.core.search.models.SearchAggregation;
+import com.adobe.cq.commerce.core.search.models.SearchResultsSet;
 
+/**
+ * Don't forget the comment
+ */
+@ProviderType
 public interface SearchResults {
 
+    /**
+     * Name of the String resource property indicating number of products to render on front-end.
+     */
+    String PN_PAGE_SIZE = "pageSize";
```

Comment:
Some of this functionality was borrowed from the `ProductList` component to allow the same authoring options (at least those that are relevant for search result pages)